src/components/feedback-form/feedback-form.js

```diff
@@ -0,0 +1,86 @@
+import Component from '../../app/js/base/Component';
+
+class FeedbackForm extends Component {
+    form;
+    sendingText;
+    warningText;
+    inputMail;
+    inputConfirm;
+    submitButton;
+    sendingWrapper;
+    formLoader;
+
+    constructor(element) {
+        super(element);
+
+        this.form = this.getElement("inner");
+        this.sendingText = this.getElement("sending-text");
+        this.warningText = this.getElement("warning");
+        this.inputMail = this.getElement("input");
+        this.inputConfirm = this.getElement("input-confirm");
+        this.submitButton = this.getElement("button");
+        this.sendingWrapper = this.getElement("sending-wrapper");
+        this.formLoader = this.getElement("loader");
+
+        this.form.addEventListener("submit", this.handlingSubmitEvent)
+    }
+
+    getFormData = () => {
+        let data = {
+            email: this.inputMail.value,
+            confirm: this.inputConfirm.checked
+        }
+        return data;
+    }
```

Comment:
Функция только формирует и возвращает объект, можно упростить запись, избавившись от переменной `data`
```suggestion
    getFormData = () => {
        return {
            email: this.inputMail.value,
            confirm: this.inputConfirm.checked
        }
    }
```